### ISSUE TITLE
Dude, Where's My Menu?

### ISSUE THREAD

**Issue by tn3rb** [TablePlus/TablePlus-Linux#176]
### Please answer the questions below, it helps us to track the issue.

1. Database version: MySQL Ver 8.0.32-0ubuntu0.22.10.2

2. TablePlus version: 0.1 (192)

3. Linux distro: Ubuntu 22.10 

4. The steps to reproduce this issue: 

   - have coworker recommend TablePlus

   - install TablePLus via these directions: https://tableplus.com/blog/2019/10/tableplus-linux-installation.html

   - try to import DB via these directions: https://docs.tableplus.com/gui-tools/import-and-export

   - look at UI and say: "Dude, Where's My Menu?"

   - look at docs to see where menu is: https://docs.tableplus.com/gui-tools/the-interface/menu

   - see this image:
     ![Screenshot from 2023-02-27 14-58-23](https://user-images.githubusercontent.com/1751030/221708894-7b5a59f8-a2f6-4a11-b969-83a1e9456a1c.png)

   - look back at UI and say: "Dude, Where's My Menu?"
     ![Screenshot from 2023-02-27 14-46-10](https://user-images.githubusercontent.com/1751030/221708983-bd789d0b-9473-4bad-b587-dc33b24fe35e.png)

Basically, what I'm trying to say is: "Dude, Where's My Menu?"

no amount of left or right clicking anywhere in the UI reveals the "File" menu, or any other menu item leaving me with only one question: "Dude, Where's My Menu?"

**Comment by ctibo** [comment]
I had the same reaction. 
I love TablePlus. I was using it a couple years ago when I was working on a Mac.
I was so happy when I learned that TablePlus has a Linux distro. The fact that the menus are nowhere to be found is a bummer though. Can't do much without menus...

**Comment by huyphams** [comment]
Hi, the menu is just replicate of the function in app. Could you tell me the feature that you're looking for?

**Comment by ctibo** [comment]
I can't seem to find the Backup & Restore features on Linux.
I think they should be under File > Backup, but I don't have a menu bar.

**Comment by mrmatteastwood** [comment]
Um yeah, same here. Linux Mint user. No menu bar.

**Comment by vrymel** [comment]
How do I even access the Restore function without the menu bar?

**Comment by huyphams** [comment]
TablePlus is not a cross platform app, so if you not find it, it is not supported @vrymel

**Comment by ipmanlk** [comment]
> TablePlus is not a cross platform app, so if you not find it, it is not supported @vrymel

Wait, So there is no point in purchasing a license for Linux? Looking at the issue tracker Linux issues seems to be neglected a lot.

**Comment by huyphams** [comment]
Why @ipmanlk?

**Comment by ipmanlk** [comment]
> Why @ipmanlk?

This page should clearly state that not all features are supported on every platform: https://tableplus.com/download. The point I was making is that when someone purchases a license after seeing the feature list, they expect to see them on their platform. Sorry if it came across as if I was discouraging paying for licenses.

**Comment by RyanPaiva56** [comment]
Did anyone figure out how to backup/restore on linux?

**Comment by jaykilleen** [comment]
Oh no... I am on Ubuntu and about to do a backup/restore with TablePlus. 
@RyanPaiva56 any chance you figured this out? I think I already know the answer. Not supported.

**Comment by ernstki** [comment]
> Did anyone figure out how to backup/restore on linux?

I can empathize with wanting a GUI for this, but [learning to use `mysqldump`](https://dev.mysql.com/doc/refman/8.4/en/mysqldump.html) will pay dividends. Maybe here's your excuse to do that.

It's also an opportunity to recalibrate your expectations for feature parity between Linux and macOS. Yes, it'd be ideal if the web site documented features that were currently unsupported for the Linux version. But this is a **native Linux app**, built from a completely independent codebase, not a web app bundled with a copy of the Chrome browser like many popular "cross-platform" desktop apps, where features can ship simultaneously for all platforms.

A lot more engineering effort is required, and it's darned impressive what they've been able to accomplish already — especially considering the state of the tooling on Linux, compared to Windows and macOS.

As far as I can tell, the Linux version is still getting routine updates. The money you paid for the Linux app helps a couple of indie developers keep doing that, while also being able to pay their bills.

**Comment by huyphams** [comment]
correct @ernstki we just figure out how to support backup/restore on Linux recently and still working on it (hint appimange)

**Comment by RyanPaiva56** [comment]
> Oh no... I am on Ubuntu and about to do a backup/restore with TablePlus. [@RyanPaiva56](https://github.com/RyanPaiva56) any chance you figured this out? I think I already know the answer. Not supported.

Yes, I switched to pgadmin 4.
